Components/Avatar/Avatar.css

```diff
@@ -0,0 +1,95 @@
+.small-avatar{
+    height: 7rem;
+    width: 7rem;
+    border-radius: 10rem;
+    margin: 1rem;
+}
+.medium-avatar{
+    height: 14rem;
+    width: 14rem;
+    border-radius: 10rem;
+    margin: 1rem;
+}
+.large-avatar{
+    height: 20rem;
+    width: 20rem;
+    border-radius: 10rem;
+    margin: 1rem;
+}
+
+.avatar-type{
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    background-color: var(--landing-hover-color);
+}
+
+iframe{
+    padding: 1rem 2rem;
+    width: 110rem;
+    height: 22rem;
+    overflow-y: scroll;
+}
+
+.carbon-container{
```

Comment:
You can create flex-center utility class. As It may need you further also.

Reply:
okay will look into it.

Reply:
added the change